assets/app/scripts/controllers/build.js

```diff
@@ -48,14 +48,6 @@ angular.module('openshiftConsole')
       $scope.logCanRun = !(_.includes(['New', 'Pending', 'Error'], build.status.phase));
     };
 
-    var updateCanBuild = function() {
```

Comment:
i think there is still a case where the a button should be disabled.  If you are looking at a build whose build config has been deleted, will Rebuild work?